server/build_event_protocol/target_tracker/target_tracker.go

```diff
@@ -112,15 +114,27 @@ func (t *target) updateFromEvent(event *build_event_stream.BuildEvent) {
 	}
 }
 
-func getLabelFromEventId(beid *build_event_stream.BuildEventId) string {
+const target_id_separator string = "|"
```

Comment:
In Bazel, for example when specifying an aspect on the command line, the separator is usually `%`.

Reply:
This is just a separator for generating an id out of a [target, aspect?] tuple--the aspect string is formatted as you describe, so it comes out as `//some/regular:target|//some/random.bzl%aspect`

Reply:
nit: Camel case?